XCUITests/SaveLoginsTests.swift

```diff
@@ -24,6 +24,22 @@ class SaveLoginTest: BaseTestCase {
         navigator.goto(LoginsSettings)
         waitForExistence(app.tables["Login List"])
     }
+    
+    func testLoginsListFromBrowserTabMenu() {
+        waitForTabsButton()
+        //Make sure you can access empty Login List from Browser Tab Menu
+        navigator.goto(LoginsBrowserTab)
+        waitForExistence(app.tables["Login List"])
+        XCTAssertTrue(app.otherElements["Enter Search Mode"].exists)
```

Comment:
Let's try to ad an id for Enter Search Mode...I'm afraid this view may change and so the test fail soon...I have looking at the code and I think it can be added after [this line](https://github.com/mozilla-mobile/firefox-ios/blob/119be2ab37cdeccb5803b323794c6f6773802c99/Client/Frontend/Widgets/SearchInputView.swift#L77) as `view.accessibilityIdentifier = "enterSearchMode"` (or the name you want :) )